chain-tx-enclave-next/mls/src/key.rs

```diff
@@ -88,6 +91,7 @@ impl Codec for HPKEPublicKey {
 
 /// p-256 private key
 /// used for obtaining the initial sealed secrets (HPKE)
+#[derive(Clone)]
 pub struct HPKEPrivateKey(<hpke::kex::DhP256 as hpke::KeyExchange>::PrivateKey);
```

Comment:
is this needed to be clone?

Reply:
removed

Reply:
The `Clone` is added back, because we need to store private keys inside tree, and tree need to be able to be cloned.